e2e/wdio/core/pages/slider.po.ts

```diff
@@ -27,7 +27,7 @@ export class SliderPo extends CoreBaseComponentPo {
     sliderLabel = '.fd-slider__label';
 
     // playground selectors
-    sliderTypeMenu = this.playgroundExamples + 'select';
+    sliderTypeMenu = this.playgroundExamples + 'select-tags.ts';
```

Comment:
what is this??

Reply:
Looks like a refactoring update issue. Fixed.